### PR TITLE
Remove `Graveyard` in favour of `Beatmap/Category#graveyard`

### PR DESCRIPTION
**Pull request by TicClick**
also some rewrite here and there. affects help centre (there's a link to `/wiki/Graveyard`), so..

### CODE REVIEW

**Comment by TicClick** [comment]
side note: I've ran into ppy/osu-web#8030, which exists for I don't know how long and affects a lot of our pages

File: wiki/Beatmap/Category/en.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
...
+Graveyard is a place for beatmaps abandoned by their [hosts](/wiki/Beatmap/Beatmap_host). Such beatmaps do not have leaderboards, but they can still be downloaded and played, and [contribute to play statistics](#unranked-beatmaps-statistics). They use the question mark icon (![](/wiki/shared/status/graveyard.png)) in the [song selection screen](/wiki/Interface#song-selection-screen).
 
-Once a beatmap is graveyarded, it can be revived (i.e. brought back to [Pending](#work-in-progress-and-pending)). This can be done by updating the beatmap via the [beatmap submission system](/wiki/Submission).
+A beatmap is moved to the Graveyard category from [Work in Progress and Pending](#work-in-progress-and-pending) automatically once it hasn't been updated for 4 weeks. The author can revive the beatmap (i.e. bring back to [Pending](#work-in-progress-and-pending)) by updating it via the [beatmap submission system](/wiki/Submission). While a beatmap is in the graveyard, it doesn't count towards a mapper's [total upload limit](/wiki/Help_Centre/Beatmapping#i-am-an-osu!supporter,-but-i-don't-have-10-pending-slots!); reviving it requires a free pending slot.
```
Comment (Walavouchey) [inline wiki/Beatmap/Category/en.md:11]:
```suggestion
A beatmap is moved to the Graveyard category from [Work in Progress and Pending](#work-in-progress-and-pending) automatically once it hasn't been updated for 4 weeks. The author can revive the beatmap (i.e. bring back to [Pending](#work-in-progress-and-pending)) by updating it via the [beatmap submission system](/wiki/Submission). A beatmap in the graveyard doesn't count towards a mapper's [total upload limit](/wiki/Help_Centre/Beatmapping#i-am-an-osu!supporter,-but-i-don't-have-10-pending-slots!), but reviving it requires a free pending slot.
```

File: wiki/Beatmap/Category/en.md [hunk 2]
```diff
@@ -6,9 +6,9 @@
 
 ### Graveyard
 
-Graveyarded beatmaps are beatmaps that have been abandoned by their creator. Beatmaps are moved to the Graveyard category from the [Work in Progress and Pending](#work-in-progress-and-pending) categories automatically once they haven't been updated for 4 weeks. Graveyarded beatmaps do not have leaderboards. However, they can still be downloaded and played, and [contribute to play statistics](#unranked-beatmaps-statistics). They use the question mark icon (![](/wiki/shared/status/graveyard.png)) in the [song selection screen](/wiki/Interface#song-selection-screen).
+Graveyard is a place for beatmaps abandoned by their [hosts](/wiki/Beatmap/Beatmap_host). Such beatmaps do not have leaderboards, but they can still be downloaded and played, and [contribute to play statistics](#unranked-beatmaps-statistics). They use the question mark icon (![](/wiki/shared/status/graveyard.png)) in the [song selection screen](/wiki/Interface#song-selection-screen).
```
Comment (Walavouchey) [inline wiki/Beatmap/Category/en.md:9]:
```suggestion
Graveyard is a place for beatmaps abandoned by their [hosts](/wiki/Beatmap/Beatmap_host). Such beatmaps do not have leaderboards, but they can still be downloaded and played, as well as [contribute to play statistics](#unranked-beatmaps-statistics). They use the question mark icon (![](/wiki/shared/status/graveyard.png)) in the [song selection screen](/wiki/Interface#song-selection-screen).
```

**Review by TPGPL — DISMISSED** [review]
lgtm

**Review by peppy — APPROVED** [review]

**Pull request merged**